### PR TITLE
Add an article about VulnerableApp

### PR DESCRIPTION
**Pull request by hexxdump**
Add an article about VulnerableApp

Fixes PR: #226 

Signed-off-by: Hussaina Begum Nandyala <hexxdump@gmail.com>

### CODE REVIEW

File: docs/blogs/WhyVulnerableApp.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Why VulnerableApp
```
Comment (preetkaran20) [inline docs/blogs/WhyVulnerableApp.md:3]: Title and file name can be changed to something else ... may be `Overview of Owasp-VulnerableApp` or if you have any other idea.

 @hemantgs  any suggestions ?

Comment (preetkaran20) [reply docs/blogs/WhyVulnerableApp.md:3]: @hexxdump  also please add the link of medium article for now to Read me ... in the last. 
Create a new Heading named Blogs and then link.

Comment (preetkaran20) [reply docs/blogs/WhyVulnerableApp.md:3]: @hexxdump thanks for adding the links also can you please rename the file and also the title as mentioned above.

Comment (hexxdump) [reply docs/blogs/WhyVulnerableApp.md]: done, pls review

File: docs/blogs/Overview of Owasp-VulnerableApp.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Why VulnerableApp
```
Comment (preetkaran20) [inline docs/blogs/Overview of Owasp-VulnerableApp.md:3]: Title is still same, can you please change this. Also can you change the name of file to `Overview_of_Owasp-VulnerableApp.md` as space in file name is little problematic.

Comment (hexxdump) [reply docs/blogs/Overview of Owasp-VulnerableApp.md]: changed title and renamed the file, pls check it @preetkaran20

**Comment by preetkaran20** [comment]
LGTM, thanks a lot.

**Review by preetkaran20 — APPROVED** [review]

**Comment by hexxdump** [comment]
> LGTM, thanks a lot.

Glad I could help!

**Pull request merged**